src/components/footer/Footer.css

```diff
@@ -41,8 +41,9 @@ footer {
 }
 
 .footer-logo{
-    width: 43px;
-    height: 45px;
+    width: 80%;
```

Comment:
Any reason we use % instead of absolute number for logo size?

Reply:
I choose the relative % value instead so as to adjust to the screen size responsively. Because when the screen gets larger (e.g. more than 1000px as below), our logo looks a little bit small if we are using the previous absolute value. 

But the difference is not very obvious. Which one would you prefer? Thanks!

![Screenshot 2023-07-16 at 12 03 20 AM](https://github.com/1thing-org/1thing.org/assets/115900322/bdc8ba62-17ee-4a20-aaea-5765f4f71c1a)
![Screenshot 2023-07-16 at 12 01 04 AM](https://github.com/1thing-org/1thing.org/assets/115900322/464353e6-827e-4e9e-a414-a828dc2fac82)